kiele-generate-report.py

```diff
@@ -220,33 +220,39 @@ def generate_static_report(report_template_path: str, reports_json_path: str, ou
     write_json_file(os.path.join(report_base_path, report_id +
                                  ".json"), json.dumps(asdict(report)))
 
-    # Create JavaScript code
-    js_code = "window.FIREFLY_REPORT_FILES = {}"
-    onlyfiles = [f for f in os.listdir(report_base_path) if os.path.isfile(os.path.join(report_base_path, f))] + [os.path.join(
-        "original", f) for f in os.listdir(os.path.join(report_base_path, "./original")) if not f.endswith(".zip")]
-    for f in onlyfiles:
-        file = open(os.path.join(report_base_path, f), "r")
-        content = file.read()
-        file.close()
-        if not f.endswith(".json"):
-            content = json.dumps(content)
-        js_code += "\nwindow.FIREFLY_REPORT_FILES[\"" + \
-            report_id + "/" + f + "\"] = " + content + "\n"
-
-    # Inject the js code into the HTML report template file.
-    f = open(report_template_path, "r")
-    template = f.read()
-    f.close()
-    report_html = template.replace(
-        "<body>", "<script>" + js_code + "</script>\n<body>")
-
-    if output_report_path.strip() == "":
-        output_report_path = report_id + ".html"
-
-    f = open(output_report_path, "w")
-    f.write(report_html)
-    f.close()
-
+    if create_report_archive == False:
+        # Create JavaScript code
+        js_code = "window.FIREFLY_REPORT_FILES = {}"
+        onlyfiles = [f for f in os.listdir(report_base_path) if os.path.isfile(os.path.join(report_base_path, f))] + [os.path.join(
+            "original", f) for f in os.listdir(os.path.join(report_base_path, "./original")) if not f.endswith(".zip")]
```

Comment:
This is a weird way to split these lines. Maybe do something like:

```
   [ f for f in ....
                      if ....
          for f in ....
                            if ...
    ]
```

Just something a litle more readable.